tools/deployment/test_torchserver.py

```diff
@@ -55,7 +55,7 @@ def align_ts_output(inputs, metainfo, device):
 
 
 def main(args):
-    register_all_modules()
+    init_default_scope('mmdet')
```

Comment:
Does it need to be unified as `init_default_scope(cfg.get('default_scope', 'mmdet'))`?